### ISSUE TITLE
Ignore file fails with no debug output

### ISSUE THREAD

**Issue by RaphaelArkadyMeyer** [divmain/GitSavvy#781]
I want to use gitsavvy to ignore a file, so trying with both the 'i' and 'I' command on the status screen, I attempted to ignore a main.pdf file. I get the following error:
\`D:\Programs and Stuff\Git\cmd\git.EXE config --global core.autocrlf\` failed with following output:

However, no error message is printed. The full call stack can be seen in the developer console:

> Traceback (most recent call last):
>   File "core.commands.ignore in C:\Users\Raphael A. Meyer\AppData\Roaming\Sublime Text 3\Installed Packages\GitSavvy.sublime-package", line 41, in on_done
>   File "core.git_mixins.ignore in C:\Users\Raphael A. Meyer\AppData\Roaming\Sublime Text 3\Installed Packages\GitSavvy.sublime-package", line 19, in add_ignore
>   File "core.git_command in C:\Users\Raphael A. Meyer\AppData\Roaming\Sublime Text 3\Installed Packages\GitSavvy.sublime-package", line 162, in git
> GitSavvy.core.exceptions.GitSavvyError: \`D:\Programs and Stuff\Git\cmd\git.EXE config --global core.autocrlf\` failed with following output:

**Comment by stoivo** [comment]
Does it work to execute this command from the shell?
```
git config --global core.autocrlf
```

What git version are you at?
Windows version

**Comment by RaphaelArkadyMeyer** [comment]
I get an empty new line response by running the above in git bash.
I have git version 2.14.1.windows.1
My windows 10 build is 10.0.15063

**Comment by mercurykd** [comment]
git 2.15.0.windows.1
Windows 10 x64 build 16299.19
sublime 3154

git ignore pattern
```
Traceback (most recent call last):
  File "core.commands.ignore in C:\Users\User\AppData\Roaming\Sublime Text 3\Installed Packages\GitSavvy.sublime-package", line 41, in on_done
  File "core.git_mixins.ignore in C:\Users\User\AppData\Roaming\Sublime Text 3\Installed Packages\GitSavvy.sublime-package", line 19, in add_ignore
  File "core.git_command in C:\Users\User\AppData\Roaming\Sublime Text 3\Installed Packages\GitSavvy.sublime-package", line 166, in git
GitSavvy.core.exceptions.GitSavvyError: `C:\Program Files\Git\cmd\git.EXE config --global core.autocrlf` failed.
```
git ignore current file
```
Traceback (most recent call last):
  File "C:\Program Files\Sublime Text 3\sublime_plugin.py", line 918, in run_
    return self.run()
TypeError: run() missing 1 required positional argument: 'file_path_or_pattern'
```

**Comment by randy3k** [comment]
It should be closed by 36c63c8.
Let's know if it doesn't.
